src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -42,11 +42,19 @@ function NameAndPicture({ navigation }: Props) {
   // CB TEMPORARY HOTFIX: Pinging Komenci endpoint to ensure availability
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
 
+  const createAccountCopyTestConfig = useSelector(createAccountCopyTestConfigSelector)
+
   useLayoutEffect(() => {
     navigation.setOptions({
       headerTitle: () => (
         <HeaderTitleWithSubtitle
-          title={t(choseToRestoreAccount ? 'restoreAccount' : 'createAccount')}
+          title={t(
+            choseToRestoreAccount
+              ? 'restoreAccount'
+              : createAccountCopyTestConfig === 'control'
```

Comment:
i think as a rule, we should check for the variants rather than the control. we're confident in showing the control (since it is the current released version) and we only want to show the variant for a specific value of remote config. now say someone makes a typing mistake, and writes "contrl" (lol) - in this case, we would accidentally show the variant. <- a bit of a contrived example but i guess the point i make is, we want to make it hard to get the variant, and everything except specific values should fall back on the control because it is safer this way

Reply:
I did as you said!